src/terra-dev-site/example/ResponsiveListAndTableExample.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react';
+import { ActiveBreakpointContext } from 'terra-application/lib/breakpoints';
+import List, { Item } from 'terra-list';
+import ItemView from 'terra-clinical-item-view';
+import Table from 'terra-table';
+import classNames from 'classnames/bind';
+import styles from './ResponsiveListAndTableExample.scss';
+import mockData from './mock-data/table-select-data';
+
+const cx = classNames.bind(styles);
+const createItem = rowData => {
+  let startAccessory;
+  let comment;
+  let endAccessory;
+
+  const displays = rowData.cells.map((cell) => {
+    switch (cell.key) {
+      case 'startAccessory':
+        startAccessory = cell.children;
+        return null;
+      case 'comment':
+        comment = cell.children;
+        return null;
+      case 'endAccessory':
+        endAccessory = cell.children;
+        return null;
+      default:
+        return cell.children;
+    }
+  });
+
+  if (startAccessory) {
+    return (
+      <Item>
+        <ItemView
+          displays={displays}
+          startAccessory={startAccessory}
+          comment={comment}
+          endAccessory={endAccessory}
+          layout={rowData.layout}
+        />
+      </Item>
+    );
+  }
+
+  return (
+    <Item>
+      <ItemView
+        displays={displays}
+        reserveStartAccessorySpace
+        comment={comment}
+        endAccessory={endAccessory}
+        layout={rowData.layout}
+      />
+    </Item>
+  );
+};
+const createItems = data => data.map(childItem => createItem(childItem));
+const listDisplay = (
+  <List role="listbox">
+    {createItems(mockData)}
+  </List>
+);
+
+const CustomItemCollection = () => {
+  const [selectedKey, setSelectedKey] = useState([]);
```

Comment:
Your state handling appears to only apply to the table version and not the list version.

Reply:
Fixed here https://github.com/cerner/terra-ui/pull/310/commits/c9207cd9bd44ec0112df2b1e223d339fd0c3e8c0#diff-db8ae280acc81345eeba3a440c2c81285c34457faf9cbfb01de5508791e3283fR68